na-editor.html

```diff
@@ -135,10 +135,11 @@
           this.__editedNoteElement.style.opacity = 1;
           this.__editedNoteElement = null;
         }
-
+        
         this.fire('close', detail, { bubbles: false });
         this.transitionOut().then(function() {
-          Polymer.dom(this).classList.remove('open');
+          var classList = Polymer.dom(this).classList || this.classList;
+          classList.remove('open');
```

Comment:
I think [`Polymer.dom(this).classList` should still work](https://github.com/Polymer/polymer/blob/master/lib/legacy/polymer.dom.html#L235); is it not there sometimes?